owncloudApp/src/main/java/com/owncloud/android/ui/adapter/LogListAdapter.kt

```diff
@@ -0,0 +1,92 @@
+/*
+ * ownCloud Android client application
+ *
+ * @author David González Verdugo
+ * Copyright (C) 2018 ownCloud GmbH.
```

Comment:
Update the year to 2019

Reply:
This complete entry is wrong
This is the right information, I would delete it from complete repo

<img width="605" alt="image" src="https://user-images.githubusercontent.com/3314607/53437630-47c3c700-39fe-11e9-9aea-beb748d45608.png">

Reply:
I guess the year is not about knowing in which moment was modified the file (git history) but copyright requirement. @michaelstingl @hosy Should we keep the year along with ownCloud Copyright in top of all the files or we can remove it? That's there from long time ago. Thanks

Reply:
The main reason of keeping this is that the source code could easily be - someday - used outside of github and the actual structure ... in which case the information would be lost.